common/crypto_utils/setup.py

```diff
@@ -63,25 +63,26 @@
 
 verify_report_include_dirs = [
     os.path.join(tcf_root_dir, 'common/cpp'),
+    os.path.join(tcf_root_dir, 'common/cpp/crypto'),
+    os.path.join(tcf_root_dir, 'common/cpp/packages/parson'),
+    os.path.join(tcf_root_dir, 'common/cpp/packages/base64'),
     os.path.join(tcf_root_dir, 'common/cpp/verify_ias_report'),
-]
+] + openssl_include_dirs
 
 library_dirs = [
     os.path.join(tcf_root_dir, "common/cpp/build"),
 ] + openssl_lib_dirs
 
 libraries = [
+    'uavalon-verify-ias-report',
+    'uavalon-crypto',
     'uavalon-common',
     'uavalon-base64',
     'uavalon-parson',
-    'uavalon-crypto',
-    'uavalon-verify-ias-report'
 ] + openssl_libs
 
-libraries = libraries + openssl_libs
-
 crypto_modulefiles = [
-    "avalon_crypto_utils/crypto/crypto.i"
+    "avalon_crypto_utils/crypto/crypto.i",
```

Comment:
Why we need a comma at the end ?

Reply:
No. Doesn't make difference it is last element the array.